sources/packages/backend/apps/queue-consumers/src/services/cas-supplier/cas-evaluation-result-processor/cas-active-supplier-not-found-processor.ts

```diff
@@ -65,8 +67,21 @@ export class CASActiveSupplierNotFoundProcessor extends CASEvaluationResultProce
       });
       summary.info("Created supplier and site on CAS.");
     } catch (error: unknown) {
-      summary.error("Error while creating supplier and site on CAS.", error);
-      return { isSupplierUpdated: false };
+      if (error instanceof CustomNamedError) {
+        if (error.name === CAS_BAD_REQUEST) {
```

Comment:
That could be one condition only:

```js
if (error instanceof CustomNamedError && error.name === CAS_BAD_REQUEST) {
```

Reply:
For other CustomNamedError other than CAS_BAD_REQUEST we need to set the summary as error and isSupplierUpdated as false.